src/SFA.DAS.ApplyService.Web/Services/ResetQnaDetailsService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.Web.Infrastructure;
+
+namespace SFA.DAS.ApplyService.Web.Services
+{
+    public class ResetQnaDetailsService : IResetQnaDetailsService
+    {
+        private readonly IQnaApiClient _qnaApiClient;
+
+        public ResetQnaDetailsService(IQnaApiClient qnaApiClient)
+        {
+            _qnaApiClient = qnaApiClient;
+        }
+
+        public async Task ResetPagesComplete(Guid applicationId, int sequenceNo, int sectionNo, string pageId)
+        {
+            await ResetSection1_5(applicationId, sequenceNo, sectionNo, pageId);
+        }
+
+        private async Task ResetSection1_5(Guid applicationId, int sequenceNo, int sectionNo, string pageId)
```

Comment:
Make this method public and get take only one parameter i.e. applicationId

Reply:
We only want the reset to happen when a specific sequence/section/page is saved for the application.  We need all the parameters, otherwise it's move all the logic in this service back into the controller, which is cluttering the controller.

Reply:
I changed this to use minimum number of parameters, which is applicationId and pageId